src/devices/video/hd44780.h

```diff
@@ -70,11 +70,13 @@ class hd44780_device : public device_t
 	enum
 	{
 		CHARSET_HD44780_A00,
+		CHARSET_HD44780U_A00,
+		CHARSET_HD44780U_A02,
 		CHARSET_SED1278_0B,
 		CHARSET_KS0066_F00,
 		CHARSET_KS0066_F05 /*,
-		CHARSET_HD44780_A01,
-		CHARSET_HD44780_A02,
+		CHARSET_HD44780U_A01,
+		CHARSET_SED1278_0A,
 		CHARSET_KS0066_F03,
 		CHARSET_KS0066_F04,
 		CHARSET_KS0066_F06,
```

Comment:
As far as I can tell, this is only used to implement `device_rom_region` the wrong way, and should go.